schemas/com.snowplowanalytics.snowplow.ecommerce/transaction/jsonschema/1-0-0

```diff
@@ -0,0 +1,78 @@
+{
+    "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+    "description": "Schema for a transaction entity in Ecommerce",
+    "self": {
+        "vendor": "com.snowplowanalytics.snowplow.ecommerce",
+        "name": "transaction",
+        "format": "jsonschema",
+        "version": "1-0-0"
+    },
+    "type": "object",
+    "properties": {
+        "transaction_id": {
+            "description": "The ID of the transaction.",
+            "type": "string",
+            "maxLength": 4096
+        },
+        "revenue" : {
+            "type": "number",
+            "description": "The revenue of the transaction.",
+            "multipleOf": 0.01,
+            "minimum": 0.00,
+            "maximum": 2147483647
+        },
+        "currency": {
+            "type": "string",
+            "description": "The currency used for the transaction (ISO 4217).",
+            "maxLength": 3,
+            "minLength": 3
+        },
+        "payment_method": {
+            "type": "string",
+            "description": "The payment method used for the transaction.",
+            "maxLength": 99
+        },
+        "total_quantity": {
+            "type": "integer",
+            "description": "Total quantity of items in the transaction.",
+            "minimum": 0
```

Comment:
Should there be a maximum on this field?

Reply:
On similar `quantity` type fields, I have not seen a maximum in the past, I can set `9999999` which is valid for our case and stays in the 32-bit range.